src/condor_io/condor_auth_ssl.cpp

```diff
@@ -356,8 +356,13 @@ int Condor_Auth_SSL::authenticate(const char * /* remoteHost */, CondorError* er
 		std::string scitoken;
 		if (m_scitokens_mode) {
 			if (m_scitokens_file.empty()) {
```

Comment:
Doesn't this say that the default scitokens file takes precedence if it exists?  I would think rather that it should be the last thing tried if none of the other options exist.

Reply:
There's no default file, actually.  This is the application-specific override.  I'd rather keep this first as it's the most specific (and preserves current behavior).